inc/utype.h

```diff
@@ -145,6 +147,35 @@ extern struct arabicforms {
     unsigned int required_lig_with_alef: 1;
 } ArabicForms[256];	/* for chars 0x600-0x6ff, subtract 0x600 to use array */
 
```

Comment:
At this point in time, I think it is best to leave that working the way it works now.
If someone is willing to explain 0600..06ff a little better, we can work at making that section more customized, but at this point it seemed to make sense to have a base to cover 'Vulgars' as originally done in the older IS_LIGATURES.c, and while searching, also expand into the other odd fractions and ligatures too.